### DISCUSSION TITLE
Which dataset did you use for text recognition model pretraining

### DISCUSSION THREAD

**Discussion by davor10105** [mindee/doctr#875] [Q&A]
As the title says, I'm just wondering which dataset did you use for any of your text recognition model pretraining, whether it is public or private, or did you generate one yourself and how?

Many thanks!

**Comment by felixdittrich92** [comment]
Hi davor10105 :wave: ,

#821 
the used dataset is private but its on track for v0.6.0 to provide more information in the documentation.
ping @fg-mindee @charlesmindee can give you more information :smiley:
